lib/node_modules/@stdlib/array/base/mskfilter-map/README.md

```diff
@@ -0,0 +1,153 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# mskfilter-map
+
+> Apply a mask to a provided input array and return a new array after applying a mapping function.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var mskfilterMap = require( '@stdlib/array/base/mskfilter-map' );
+```
+
+#### mskfilterMap( x, mask, clbk\[, thisArg] )
+
+Returns a new array by applying a mask to a provided input array after applying mapping function.
+
+```javascript
+var x = [ 1, 2, 3, 4 ];
+
+function customMapping( value ) {
+    return value * 2;
+}
+
+var y = mskfilterMap( x, [ 0, 1, 0, 1 ], customMapping );
+// returns [ 4, 8 ]
+```
+
+The function supports the following parameters:
+
+-   **x**: input array.
+-   **mask**: mask array.
+-   **clbk**: function to apply.
+-   **thisArg**: applied function execution context (_optional_).
+
+To set the applied function's execution context, provide a `thisArg`.
```

Comment:
You should provide an example here of providing a `thisArg`. A common example is counting the number of times the callback is invoked. You can copy the example above and then provide a context object which gets incremented after each invocation. For the example above, `clbk` should be called twice. See other base array packages.

Reply:
@AuenKr I believe the README is still missing this example.

Reply:
I see that you added it below in the `assign` API. I'd go ahead and move (and modify) that up here.